### DISCUSSION TITLE
Reading .xlsx file

### DISCUSSION THREAD

**Discussion by Polina418** [jspsych/jsPsych#1395] [Q&A] [answered]
Hello! 
I'm working on a naming task and I can't figure out how to read stimuli from a file(.xlsx), that contains 2 columns (image name and a condition). The only possible way I could find is to use PsychoJS´s TrialHadler, but I can't make it work. Here is an example of my .xlsx file and a part of the code for using the TrialHadler in case it can help. Any help, hints will be much appreciated! 

Biling_Switch_old.xlsx
image1.bmp   11
image2.bmp   12
image3-bmp  11

...
'''      bili_trials = new TrialHandler({
        psychoJS: psychoJS,
        nReps: 1, method: TrialHandler.Method.SEQUENTIAL,
        extraInfo: expInfo, originPath: undefined,
        trialList: 'Biling_Switch_old.xlsx',
        seed: undefined, name: 'bili_trials'
      });
'''

**Comment by jodeleeuw** [comment, marked as answer]
Hi @Polina418 ,

I [recorded a video tutorial](https://www.youtube.com/watch?v=T3GCZWarEsY) that covers how to take trial information from a spreadsheet and convert it into a format that jsPsych can use.

**Reply by Polina418** [reply]
Hi @jodeleeuw. The video is super useful.  Thank you so much, you saved my day!
